packages/extensions/openapi-to-typespec/test/utils/generate-typespec.ts

```diff
@@ -67,6 +93,9 @@ async function main() {
     const folder = folders[i];
     try {
       await generateTypespec(repoRoot, folder, debug, i % 2 === 0);
+      if (swagger) {
```

Comment:
Is that not going to crash if they don’t have installed typespec and the dependencies yet?

Reply:
I don't expect users to call it. But if you want, I could move the dev dependency to dependency. Is that what you are saying?

Reply:
Oh no is that just meant as test in this repo?

Reply:
It's just test purpose.